lib/docs/stories/introduction/usage.stories.mdx

```diff
@@ -1,42 +1,63 @@
+- Code editor of your choice (e.g. [Visual Studio Code](https://code.visualstudio.com/download))
+
+## Steps
+
+Let's get started!
+
+### Create a PubNub account
+
+Before you start, you need to get **Publish Key** and **Publish Key** for your chat app.
+Sign in or create an account on the [Admin Portal](https://dashboard.pubnub.com/) to create an app and a new keyset. 
 
-## Set up PubNub account
+>**TIP**: When you create a new app on the Admin Portal, the first set of keys is generated automatically, but a single app can have as many keysets as you like. We recommend that you create separate keysets for production and test environments. There are some of the functionalities you might need to enable on your keyset depending on the use case, but they won't be required to complete this Getting Started guide. These options include:
+   - [**PRESENCE**](https://www.pubnub.com/docs/sdks/javascript/api-reference/presence) to monitor the subscribers of channels and delivers information on their real-time status
+   - [**STORAGE & PLAYBACK**](https://www.pubnub.com/docs/sdks/javascript/api-reference/storage-and-playback) (including correct **Retention**) to persist all messages as they're published.
+   - [**OBJECTS**](https://www.pubnub.com/docs/sdks/javascript/api-reference/objects) to easily store metadata about your application users and channels, and their membership associations, without the need to stand up your own databases. When using this feature, make sure you select a geographical region corresponding to most users of your application and to have these enabled: **User Metadata Events**, **Channel Metadata Events**, and **Membership Events**.
 
-1. Sign in or create an account to create an app on the
-   [Admin Portal](https://dashboard.pubnub.com/) and get the keys to use in your application.
+### Create a new project for your app
 
-2. When you create a new app, the first set of keys is generated automatically, but a single app can
-   have as many keysets as you like. We recommend that you create separate keysets for production
-   and test environments.
+Create a sample React project you will use as a placeholder for your chat app code. 
+To use the [`react-sample-chat`](https://www.pubnub.com/docs/sdks/javascript/react/#download-the-react-wrapper) app, follow these steps:
 
-3. Some of the functionalities you might want to enable on your keyset depending on the use-case
-   include _Presence_, _Storage & Playback_ (including correct Retention) and _Objects_ (be sure to
-   select a geographical region corresponding to most users of your application and to enable User,
-   Channel and Membership Events).
+```js
+npx create-react-app react-sample-chat
+cd react-sample-chat
+```
 
-## Install and use components
+### Install components
 
-1. Install the components and all required dependencies using npm.
+Add the PubNub JavaScript SDK, React framework packages, and PubNub Chat Components for React:
```

Comment:
I would switch "React framework packages" to "PubNub React SDK", React framework itself is already pre-installed when using `create-react-app`